website/templates/website/header.html

```diff
@@ -18,22 +18,31 @@
 
     <!-- Collect the nav links, forms, and other content for toggling -->
     <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-      <ul class="nav navbar-nav navbar-right websiteNavbarList">
-        <li><a href="/dashboard">Admin Panel</a></li>
-      </ul>
-      <ul class="nav navbar-nav navbar-right websiteNavbarList">
-        <li><a href="/cite">Cite</a></li>
-      </ul>
-      <ul class="nav navbar-nav navbar-right websiteNavbarList">
-        <li><a href="http://ghoshbishakh.github.io/dipy/newTheme/documentation.html" target="_blank">Documentation</a></li>
-      </ul>
-      {% for page in pages_in_nav %}
-      <ul class="nav navbar-nav navbar-right websiteNavbarList">
-        <li><a href="/page/{{page.website_position_id}}">{{page.title}}</a></li>
-      </ul>
-      {% endfor %}
-      <ul class="nav navbar-nav navbar-right websiteNavbarList">
-        <li><a href="/">Home</a></li>
+      <ul class="nav navbar-nav navbar-right">
+        <li class="websiteNavbarList"><a href="/">Home</a></li>
+        {% for page in pages_in_nav %}
+        <li class="websiteNavbarList"><a href="/page/{{page.website_position_id}}">{{page.title}}</a></li>
```

Comment:
Instead of using the "websiteNavbarList" class on each .li elements, why don't you just give an id to the containing ul, and, in the CSS, apply the same styles defined in the websiteNavbarList style to li elements that are part of the navbar IDed ul?